src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -108,9 +108,10 @@ public ImmutableArray<PackageSource> GetPackageSources()
                 packageSources = _packageSourceProvider.Value.GetSources(includeUnOfficial: true, includeDisabled: false)
                     .SelectAsArray(r => new PackageSource(r.Key, r.Value));
             }
-            catch (Exception ex) when (ex is InvalidDataException || ex is InvalidOperationException)
+            catch (Exception ex) when (ex is InvalidDataException || ex is InvalidOperationException || ex is ArgumentException)
             {
                 // These exceptions can happen when the nuget.config file is broken.
+                FatalError.ReportWithoutCrash(ex);
                 packageSources = ImmutableArray<PackageSource>.Empty;
             }
```

Comment:
can you add a new catch?   the existing exceptions are known and have lots of issues already reported for them.  we don't need them rereported.  this new issue seems important so we can get data on how many times it is happening and push for teh nuget team to fix it.  thanks!